utilities/install.py

```diff
@@ -1,49 +1,94 @@
-
-# print("Enabling Geppetto NetPyNE Extension ...")
-# jsonFile = open(
-#     '../org.geppetto.frontend.jupyter/src/jupyter_geppetto/geppetto/src/main/webapp/extensions/extensionsConfiguration.json',
-#     "w+")
-# jsonFile.write(json.dumps({"geppetto-netpyne/ComponentsInitialization": True}))
-# jsonFile.close()
-
-# Installing and building
-print("NPM Install and build for Geppetto Frontend  ...")
-subprocess.call(['npm', 'install'], cwd='../org.geppetto.frontend.jupyter/src/jupyter_geppetto/geppetto/src/main/webapp/')
-subprocess.call(['npm', 'run', 'build-dev-noTest'], cwd='../org.geppetto.frontend.jupyter/src/jupyter_geppetto/geppetto/src/main/webapp/')
-
-print("Installing jupyter_geppetto python package ...")
-subprocess.call(['pip', 'install', '-e', '.'], cwd='../org.geppetto.frontend.jupyter')
-print("Installing jupyter_geppetto Jupyter Extension ...")
-subprocess.call(['jupyter', 'nbextension', 'install', '--py', '--symlink', '--user', 'jupyter_geppetto'], cwd='../org.geppetto.frontend.jupyter')
-subprocess.call(['jupyter', 'nbextension', 'enable', '--py', '--user', 'jupyter_geppetto'], cwd='../org.geppetto.frontend.jupyter')
-subprocess.call(['jupyter', 'nbextension', 'enable', '--py', 'widgetsnbextension'], cwd='../org.geppetto.frontend.jupyter')
-subprocess.call(['jupyter', 'serverextension', 'enable', '--py', 'jupyter_geppetto'], cwd='../org.geppetto.frontend.jupyter')
-
-print("Installing NetPyNE UI python package ...")
-subprocess.call(['pip', 'install', '-e', '.'], cwd='..')
+from setuptools.command.install import install
+
+def checkDependencies():
+    '''checksifnpmisinstalled'''
+    shell = (sys.platform == 'win32')
+    try:
+        subprocess.check_call(['npm', '--version'], shell=shell)
+        subprocess.check_call(['node', '--version'], shell=shell)
+    except:
+        print('ATENTION: -> -nodejs- -npm- not found... Please visit:')
+        print('https://nodejs.org')
+        return False
+
+    try:
+        subprocess.check_call(['git', '--version'], shell=shell)
+        return True
+    except:
+        print('ATENTION: -> "git" not found... You ca get it by:')
+        print('sudo apt upgrade')
+        print('sudo apt install git')
+        return False
+
+
+if checkDependencies():
+
+    # Update pip
+    subprocess.call(["pip", "install", "--upgrade", "pip", "--user"])
+
+    # Find current directory
+    here = os.path.join(os.path.dirname(os.path.abspath(__file__)))
+    here = os.path.dirname(here)
+
+    # Install dependency - editable mode
+    print("Cloning PyGeppetto...")
+    http = 'git+https://github.com/openworm/pygeppetto.git@development#egg=pygeppetto'
+    subprocess.call(['pip', 'install', '-e', http, '--src', os.path.join(here), "--user"])
+
+    # Install dependency - editable mode
+    print("Cloning NetPyNE...")
+    http = 'git+https://github.com/Neurosim-lab/netpyne.git@metadata#egg=netpyne'
+    subprocess.call(['pip', 'install', '-e', http, '--src', os.path.join(here), "--user"])
```

Comment:
@frodriguez4600 You have changed the way netpyne and pygeppetto are installed. If I am not wrong you are not cloning those two repos anymore, right? I am not sure if this is the right approach. While developing, at the moment, it would be useful to be able to modify the sources of these repos. What do you think?

Reply:
You are right, probably it would be better if I am already `checkout` in `development`. That way, we only have to do `git remote set-url` and change to our fork.

Right now, if I want to `push` or `pull`, I would have to go inside "/pygeppetto" and type `git checkout development`. Then I can follow from there on. 

I will rollback on that, and I PR again. Thanks @adrianq

Reply:
You can just update this branch and it will update the PR. Btw, I have just changed sth in the install script so remember to pull the latest from development. It is just a couple of lines so shouldn't be too difficult to merge it